test/client/spec/components/measure-text.spec.jsx

```diff
@@ -0,0 +1,100 @@
+// Help PhantomJS out
+require("babel-polyfill");
+
+import measureText from "../../../../src/index";
+
+class MockCtx {
+  measureText() {
+    return { width: 42 };
+  }
+}
+
+class MockCanvas {
+  getContext() {
+    return new MockCtx();
+  }
+}
+
+describe("components/measure-text", () => {
+  it("should measure a single line of text", () => {
+    const measurement = measureText({
+      text: "The quick brown fox jumps over the lazy dog",
+      fontFamily: "Helvetica Neue",
+      fontSize: "24px",
+      lineHeight: "1.2",
+      fontWeight: 400,
+      fontStyle: "normal",
+      canvas: new MockCanvas()
+    });
+
+    expect(measurement).to.have.deep.property("width", "42px");
+    expect(measurement).to.have.deep.property("height", `${24 * 1.2}px`);
+  });
+
+  it("should measure all lines of text in an array", () => {
+    const measurement = measureText({
+      text: [
+        "The quick brown fox jumps over the lazy dog",
+        "The lazy fox jumps over the quick brown dog",
+        "The dog jumps over the quick, lazy brown fox"
+      ],
+      fontFamily: "Helvetica Neue",
+      fontSize: "24px",
+      lineHeight: "1.2",
+      fontWeight: "bold",
+      fontStyle: "italic",
+      canvas: new MockCanvas()
+    });
+    expect(measurement).to.have.deep.property("width", "42px");
+    expect(measurement).to.have.deep.property("height", `${(24 * 1.2) * 3}px`);
+  });
+
+  it(`should calculate height when provided a
+      px font size and a unitless line height`, () => {
```

Comment:
Isn't this multiline?
https://github.com/FormidableLabs/measure-text/pull/1/files#diff-53e8a54daecd2cfc0945fac43cce96e7R36